src/client/content_cso.cpp

```diff
@@ -55,7 +55,7 @@ class SmokePuffCSO: public ClientSimpleObject
 		MapNode n = env->getMap().getNode(floatToInt(pos, BS), &pos_ok);
 		light = pos_ok ? decode_light(n.getLightBlend(env->getDayNightRatio(),
 							env->getGameDef()->ndef()->getLightingFlags(n)))
-		               : 64;
+							: 64;
```

Comment:
```suggestion
						: 64;
```
IMO, the previous alignment was better, since the line above is part of the function call two lines above.
Alternately, for compactness, `: 64;` could be put directly at the end of the line above, or the ternary operator could be split properly, but then it would get too big.

(Also, according to our code style guildlines we only add two tabs for those types of line breaks.)

Reply:
I will have to add some style of exceptions if we want indents to be checked and we want to allow exceptions at the same time.

But in that case, the list of exceptions has to be kept up to date.

Reply:
> IMO, the previous alignment was better ...

I meant the alignment not the indents.
You replaced 15 spaces with 5 tabs.
I want it to be 4 tabs instead.

I don't get why this should be an exception.

Reply:
Yes, I overindent it. 

My comment was related to "IMO, the previous alignment was better, since the line ..."